test/Jamfile

```diff
@@ -257,6 +275,7 @@ project
     [ run functional/invoke_procedure.cpp ]
     [ run sequence/swap.cpp ]
 
+    [ run support/config.cpp ]
```

Comment:
Add `<test-info>always_show_run_output` here, like this: https://github.com/boostorg/pfr/blob/boost-1.81.0/test/Jamfile.v2#L52